test/test_openapi.py

```diff
@@ -120,6 +121,67 @@ def route2():
                 pass
         self.assertEqual(str(cm.exception), 'Only specify path_spec once per route path')
 
+    def test_shared_path_spec(self):
+        app = AzulChaliceApp('testing', spec={'foo': 'bar'})
+        swagger_test_path_spec = {
+            'parameters': [
+                params.query('foo', schema.optional({'type': 'string'})),
+            ]
+        }
+
+        @app.route('/swagger-test-1', methods=['GET'], cors=True, path_spec=swagger_test_path_spec, method_spec={
+            'summary': 'Swagger test 1',
+            'description': 'Swagger test 1',
+            'responses': {'201': {'description': 'OK'}}
+        })
+        def swagger_test_1():
+            pass
+
+        @app.route('/swagger-test-2', methods=['GET'], cors=True, path_spec=swagger_test_path_spec, method_spec={
+            'summary': 'Swagger test 2',
+            'description': 'Swagger test 2',
+            'responses': {'202': {'description': 'OK'}}
+        })
+        def swagger_test_2():
+            pass
+
+        expected_spec = {
+            'foo': 'bar',
+            'paths': {
+                '/swagger-test-1': {
+                    'parameters': [{
+                        'name': 'foo',
+                        'in': 'query',
+                        'required': False,
+                        'schema': {'type': 'string'}
+                    }],
+                    'get': {
+                        'summary': 'Swagger test 1',
+                        'description': 'Swagger test 1',
+                        'responses': {
+                            '201': {'description': 'OK'}
+                        }
+                    }
+                },
+                '/swagger-test-2': {
```

Comment:
Duplicative.

Reply:
I'm interpreting this comment to mean that I should consolidate this JSON into a dict comprehension.